src/components/Related/DomainEntriesOnStructure/index.tsx

```diff
@@ -318,8 +337,8 @@ const EntriesOnStructure = ({
 
           return (
             <div key={i} className={css('vf-stack')}>
-              <h4 id={`protvista-${e.chain}`}>
-                Chain {e.chain}
+              <h4 id={`protvista-${chainsData.chain}`}>
+                Chain{chainsList.length > 1 && 's'} {chainsList.join(',')}
```

Comment:
Show

> Chains: A, B, C

instead of

> Chains A,B,C

```suggestion
                Chain{chainsList.length > 1 && 's'}: {chainsList.join(', ')}
```